functions/createOrder/source.js

```diff
@@ -45,9 +45,9 @@ exports = function(orderDetails){
     if (arr.length !== 0 && orderDetails.type == 'REQUEST'){
```

Comment:
Will this put a freeze on requests after submitting a donation? I'm just curious since it's checking the length of the results and querying by address, if I submit a donation first and then try to submit a request, i think it may block it.